src/models/RequestActions.php

```diff
@@ -29,6 +29,10 @@ class RequestActions implements RestInterface
 {
     use SetIdTrait;
 
+    /** timeout in minutes before an active action can be requested again */
+    // ToDo?: make it a configuration on team/instance level
+    public const int RE_REQUEST_TIMEOUT = 15;
```

Comment:
I don't really understand the timeout approach.

What do we want to prevent? If a user requests the same action for the same target user, just throw an error with an informative message that this action has already been requested. Now what would be useful is to have a way to "ping" the target user, so it would send the notification email again, but that's for later.

IMHO a timeout doesn't make sense in this context.

Reply:
While I was testing, I noticed that you could request the same action over and over again. This could be used to spam a target user. I did not want to remove the possibility to request the same action again completely, but I wanted to prevent the possibility to spam.
I agree that a reminder/ping message would work as well, but then again, without a timeout, a target user could be spammed by the requester.

Reply:
>   I did not want to remove the possibility to request the same action again completely

But why though? There is no need to allow having the same action for the same target from the same requester. I'd suggest removing the possibility to make the same action. And removing the timeout code of course.

Reply:
Okay, I will remove the timeout and prevent requesters from requesting the same action again.